components/collective-navbar/NavBarCategoryDropdown.js

```diff
@@ -178,23 +180,34 @@ NavBarCategory.propTypes = {
 
 const NavBarCategoryDropdown = ({ useAnchor, collective, category, isSelected, links }) => {
   const displayedLinks = links.filter(link => !link.hide);
+
+  // eslint-disable-next-line react/prop-types
+  const NavBarScrollContainer = ({ children }) =>
+    useAnchor ? (
+      <Scrollchor to={`#category-${category}`}>{children}</Scrollchor>
+    ) : (
+      <React.Fragment>{children}</React.Fragment>
+    );
+
```

Comment:
Good approach! Some small things we can probably improve here:
1. Let's not re-define the component at each render, we can move it outside of `NavBarCategoryDropdown`. Why? For the same reason as https://reactjs.org/docs/higher-order-components.html#dont-use-hocs-inside-the-render-method.
2. Do we really need to wrap `children` in `React.Fragment`?
3. Can we avoid disabling the proptypes validation?

```suggestion
  const NavBarScrollContainer = ({ useAnchor, category, children }) =>
    useAnchor ? (
      <Scrollchor to={`#category-${category}`}>{children}</Scrollchor>
    ) : (
      children
    );
```

Reply:
Thanks. I've moved this outside of the `NavBarCategoryDropdown` and added proptypes validation. 👍🏽